### ISSUE TITLE
Active Line display on Kanban board

### ISSUE THREAD

**Issue by ScottKillen** [kepano/obsidian-minimal#758]
### Is the bug present when using the default Obsidian theme?

No

### Is the bug present when snippets and plugins are disabled?

No

### Minimal theme version

7.7.4

### Describe the bug

When using a Kanban board from the supported Kanban plugin, the Active Row is styled awkwardly when adding a card is styled. Likely, the active row should not be styled on a Kanban board.

![image](https://github.com/kepano/obsidian-minimal/assets/1509989/60b7b344-54d6-44af-9d8d-0bdecf2913dd)

### Debug info

SYSTEM INFO:
	Obsidian version: v1.6.4
	Installer version: v1.6.3
	Operating system: Windows 10 Pro 10.0.22631
	Login status: logged in
	Catalyst license: vip
	Insider build toggle: on
	Live preview: on
	Base theme: dark
	Community theme: Minimal v7.7.4
	Snippets enabled: 75
	Restricted mode: off
	Plugins installed: 38
	Plugins enabled: 38
		1: Templater v2.3.3
		2: Minimal Theme Settings v8.0.2
		3: Style Settings v1.0.8
		4: MySnippets v1.2.3
		5: Tasks v7.5.0
		6: Calendar v1.5.10
		7: Dataview v0.5.67
		8: Natural Language Dates v0.6.2
		9: Git v2.24.3
		10: BRAT v1.0.1
		11: Open vault in VSCode v1.2.2
		12: Pane Relief v0.5.3
		13: Recent Files v1.4.1
		14: Charts v3.9.0
		15: Kanban v2.0.51
		16: Commander v0.5.1
		17: Quick Explorer v0.2.8
		18: Plugin Update Tracker v1.5.2
		19: Meta Bind v1.1.2
		20: Hotkeys for specific files v1.4.1
		21: MetaEdit v1.8.2
		22: HelpMate v1.0.6
		23: Settings Search v1.3.10
		24: Iconize v2.13.0
		25: Hover Editor v0.11.20
		26: Note Gallery v0.0.59
		27: Text Transporter v1.0.11
		28: WordNet Dictionary v2.0.3
		29: Multi Properties v1.1.1
		30: File Color v1.1.0
		31: Auto Link Title v1.5.4
		32: Callout Manager v1.1.0
		33: Jump-to-Date v1.0.16
		34: List Callouts v1.2.7
		35: Canvas CSS class v2.5.0
		36: File Explorer++ v1.2.1
		37: Mononote v1.2.2
		38: Tag Wrangler v0.6.1

RECOMMENDATIONS:
	Custom theme and snippets: for cosmetic issues, please first try updating your theme and disabling your snippets. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community theme and snippets.
	Community plugins: for bugs, please first try updating all your plugins to latest. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community plugins.

**Comment by kepano** [comment]
I am not able to reproduce the issue. Please try again in a sandbox vault, or with plugins/snippets disabled

**Comment by chaosarium** [comment]
I encountered the same issue and was able to reproduce it in a sandbox vault.

![CleanShot 2024-07-07 at 18 59 22@2x](https://github.com/kepano/obsidian-minimal/assets/38693485/e22ebed9-1da5-47f9-831f-e2ddc7c18802)

Debug info:

```
SYSTEM INFO:
	Obsidian version: v1.6.5
	Installer version: v1.4.13
	Operating system: Darwin Kernel Version 23.5.0: Wed May  1 20:16:51 PDT 2024; root:xnu-10063.121.3~5/RELEASE_ARM64_T8103 23.5.0
	Login status: not logged in
	Insider build toggle: off
	Live preview: on
	Base theme: adapt to system
	Community theme: Minimal v7.7.5
	Snippets enabled: 0
	Restricted mode: off
	Plugins installed: 3
	Plugins enabled: 3
		1: Minimal Theme Settings v8.0.2
		2: Kanban v2.0.51
		3: Style Settings v1.0.8

RECOMMENDATIONS:
	Custom theme and snippets: for cosmetic issues, please first try updating your theme and disabling your snippets. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community theme and snippets.
	Community plugins: for bugs, please first try updating all your plugins to latest. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community plugins.

```

**Comment by kepano** [comment]
Fixed in 7.7.6
